src/util/net/util.h

```diff
@@ -0,0 +1,43 @@
+#ifndef HEADER_fd_src_util_net_util_h
+#define HEADER_fd_src_util_net_util_h
+
+#include "fd_udp.h"
+#include "fd_eth.h"
+
+typedef struct __attribute__((packed)) {
+  fd_eth_hdr_t eth[1];
+  fd_ip4_hdr_t ip4[1];
+  fd_udp_hdr_t udp[1];
+} eth_ip_udp_t;
```

Comment:
This definitely needs to be renamed, since it's no longer a static thing. The function needs to be renamed as well.